torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1678,6 +1694,31 @@ def _make_tensor_helper(shape, low=None, high=None):
                        SkipInfo('TestCommon', 'test_variant_consistency_jit',
                                 device_type='cuda', dtypes=[torch.float16]),
                    )),
+    OpInfo('cumsum',
+           dtypesIfCPU=all_types_and_complex_and(torch.bool),
+           dtypesIfCUDA=all_types_and_complex_and(torch.bool, torch.half),
+           skips=(
+               # For integer inputs,
```

Comment:
Hey @kshitij12345 good catch! could you file an issue for this? I am not sure if this is the desired behavior. But even so, it's not documented so we could repurpose the issue to either fix this behavior or update the doc!

Reply:
Will open an issue. Thanks!

Reply:
thanks and could you also link that issue in a comment above the skip?